src/bridge/mod.rs

```diff
@@ -85,6 +85,16 @@ async fn start_neovim_runtime() {
     let nvim = Arc::new(nvim);
 
     start_ui_command_handler(nvim.clone());
+
+    // Verify if the files to open is more than one
+    if settings.target_files.len() > 1 {
```

Comment:
Pretty sure this check can be omitted, because `.skip` will then just return an empty iterator.

Reply:
I just checked and it seems that the -p parameter does the same as my PR :c

Reply:
And I think that `.skip` is a good option, thank you very much.

Reply:
The if has been removed, and regarding the -p option, I think this PR would be more of a shortcut on the neovide side